conpot/tests/helpers/s7comm_client.py

```diff
@@ -275,6 +275,26 @@ def Function(self, _type, group, function, data=''):
             raise S7Error(code)
         return response.data[4:]
 
+    def plc_stop_function(self):
+
+        pdu_type = 1
+        request_id = 256
+        stop_func_parameter = struct.pack('!B5x10p',
+                                                    0x29               # function code
+                                                                       # Padding
+                                          , str_to_bytes('P_PROGRAM')  # Function Name
+
+                                          )
```

Comment:
3 lines of whitespace padding?
Please also note the code style we used throughout the project :)

Reply:
@creolis Sorry about that. Fixed it 😅